step/step.go

```diff
@@ -0,0 +1,121 @@
+package step
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/bitrise-steplib/bitrise-step-android-instrumented-test/apk_info"
+
```

Comment:
unnecessary newline

Reply:
This seems to be convention in our other Steps, to have a newline between the internal and external dependencies. I can remove it if you insist, but I do personally like having them separated. Just a little more organized.

Reply:
You are right, this is my mistake, for some reason I have seen 2 newlines between the std and 3rd party libs.

Reply:
Gotcha, no worries!